nengo_gui/gui.py

```diff
@@ -100,9 +105,9 @@ def _confirm_shutdown(self, signum, frame):
             if line[0].lower() == 'y':
                 raise ServerShutdown()
             else:
-                print("Resuming...")
+                logger.info("Resuming...")
         else:
-            print("No confirmation received. Resuming...")
```

Comment:
These last two `print` statements should either stay `print`s or should be converted to `sys.stdout.write` because this output is part of direct user interaction and should never be redirected to a log file for example.